packages/core/config/core-config-server-internal/tsconfig.json

```diff
@@ -14,5 +14,5 @@
   },
   "include": [
     "src/**/*"
-  ]
+, "../../test-helpers/core-test-helpers-deprecations-getters/src/test_utils.ts"  ]
```

Comment:
Q: Do we need this? AFAIK, it's an "npm" dependency, so we don't need to reference it like that. Do we?

Reply:
I didn't notice this, this is referencing a file outside of the package, which isn't going to be available in bazel. We should just be able to import the `@kbn/core-test-helpers-deprecations-getters` package in the code and be fine

Reply:
oops, that's what you get for late-day POC PR's!